elasticsearch/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG ELK_VERSION
+
+# https://www.docker.elastic.co/
+FROM docker.elastic.co/elasticsearch/elasticsearch:${ELK_VERSION}
+
+# Add your elasticsearch plugins setup here
+# Example: RUN elasticsearch-plugin install analysis-icu
```

Comment:
Do we need this folder? If so, is it only used to host the dockerfile of ES? 
What are the other files used for in the analysis folder?

Reply:
The files in the analysis folder are used to recognize synonyms.
So the folder is used both for the Dockerfile and the synonym config.